File: C++ Programming/problems on numberical/problem71.cpp
/*

71. Accept N numbers from user and display all such elements which are even and divisible by 5. 
	
	Input :  N :   6 
	Elements : 85 66 3 80 93 88  
	Output : 85 80   
	
	Author : Milind Nalawade
	
	Date : 07/07/2021
	
	Language : C++ programming
	
*/
using namespace std;
#include<iostream>
#include<stdlib.h>

//Logic

void Output(int* p,int size)
{
	cout<<"\n Nunber divisible by 5 are :- \n";
	int i=0;
	while( i!=size)
	{	
		if(p[i]%5==0 && p[i]%2==0)
		{
			cout<<" "<<p[i]<<" ";
			i++;
		}
		else
		{
			i++;
		}
	}
}

//Entry point function

int main()
{	
	int size=0;
	cout<<"\n Enter number of element : ";
	cin>>size;
	int* p=(int *)malloc(sizeof(int));
	if(p==NULL)
	{
		cout<<" Memory not avalible ";
	}
	for(int i=0;i<size;i++)
	{
	    cout<<"\n";
		cout<<" Enter "<<i+1 <<" element : ";
		cin>>p[i];
		cout<<"\n";
	}
	Output(p,size);
	free(p);
	return 0;
} 	

File: Java Programming/problems on numbericals/problem71.java
/*

71. Accept N numbers from user and display all such elements which are even and divisible by 5. 
	
	Input :  N :   6 
	Elements : 85 66 3 80 93 88  
	Output : 85 80   
	
	Author : Milind Nalawade
	
	Date : 07/07/2021
	
	Language : Java 14.0.2
	
*/

import java.lang.*;
import java.util.*	;


class Logic
{
		public static void Output(int arr[])
		{
			System.out.println("\n numbers divisible by 5 are : \n");
			for(int j=0;j<arr.length;j++)
			{
				if(arr[j]%5==0 && arr[j]%2==0)
				{	
					System.out.println(arr[j]);
				}	
			}
		}
		
}

class problem71 extends Logic
{
	public static void main(String args[])
	{
		Scanner sobj =new Scanner(System.in);
		System.out.print("\n how many number to accept : ");
		int size=sobj.nextInt();
		int arr[]=new int[size];
		System.out.println();
		for(int i=0;i<size;i++)
		{
			System.out.print(" Enter "+(i+1)+" element : " );
			arr[i]=sobj.nextInt();
			System.out.println();
		}
		Output(arr);
	}
}